web/src/components/profileEdit/QuotaCardEdit.tsx

```diff
@@ -131,35 +155,53 @@ export const QuotaCardEdit: React.FC<IQuotaCardEditProps> = (props) => {
     ) {
       return DEFAULT_QUOTA_INFO;
     }
+
     return {
-      cpuNums: quotaSizes[selectedSizes.quotaCpuSize]?.cpuNums || [],
-      memoryNums: quotaSizes[selectedSizes.quotaMemorySize]?.memoryNums || [],
-      storageNums: quotaSizes[selectedSizes.quotaStorageSize]?.storageNums || [],
-      snapshotNums: quotaSizes[selectedSizes.quotaSnapshotSize]?.snapshotNums || [],
+      cpuNums:
+        quotaSizes.CPU_QuotaSize_Detail.find(
+          (cpuOption: CPUQuotaSizeDedetail) => cpuOption.name === selectedSizes.quotaCpuSize,
+        )?.cpuNums || [],
+      memoryNums:
+        quotaSizes.Memory_QuotaSize_Detail.find(
+          (memoryOption: MemoryQuotaSizeDedetail) =>
+            memoryOption.name === selectedSizes.quotaMemorySize,
+        )?.memoryNums || [],
+      storageNums:
+        quotaSizes.Storage_QuotaSize_Detail.find(
+          (storageOtion: StorageQuotaSizeDedetail) =>
+            storageOtion.name === selectedSizes.quotaStorageSize,
+        )?.storageNums || [],
+      snapshotNums:
+        quotaSizes.Snapshot_QuotaSize_Detail.find(
+          (snapshotOption: SnapshotQuotaSizeDedetail) =>
+            snapshotOption.name === selectedSizes.quotaSnapshotSize,
+        )?.snapshotNums || [],
```

Comment:
not awesome. Also shouldn't these values be memoized? `useMemo`

Reply:
{
  cpu:
     name:
     size:
   memory:
     name:
     size:

}

instead of <object>Nums

why don't use call it `size` or `amount` or `count` ?